tasks/build.py

```diff
@@ -136,13 +126,13 @@ def create_pr_dir(pr, jobs_base_dir, event_info):
     pr_id = 'pr_%s' % pr.number
     event_id = 'event_%s' % event_info['id']
     event_dir = os.path.join(jobs_base_dir, ym, pr_id, event_id)
-    mkdir(event_dir)
+    os.makedirs(event_dir, exist_ok=True)
```

Comment:
Using `os.makedirs` makes total sense. This wasn't supported yet in Python 2.7, so I guess force of habit made me go with the usual `mkdir` approach :)